src/assets/svgs/FlowerSuccessSvg.js

```diff
@@ -1,15 +1,88 @@
 import React from 'react';
 import { Svg, Path } from 'react-native-svg';
+import { themes } from 'constants/styleGuide';
 
-export default () => (
-  <Svg width="57" height="97" viewBox="0 0 57 97" fill="none" xmlns="http://www.w3.org/2000/svg">
-  <Path d="M10.1775 18.562C12.0297 16.4081 15.1136 16.4267 17.4103 18.432C22.4761 22.8698 22.828 29.2293 22.6798 29.4522C22.5316 29.675 16.1415 30.1763 11.0573 25.7385C10.0023 24.8962 9.31908 23.6725 9.15456 22.3305C8.99004 20.9885 9.35732 19.6354 10.1775 18.562Z" fill="#2BD67B" />
-  <Path d="M56.1486 41.8369C54.1111 38.1233 49.6659 37.1392 45.6559 39.3302C36.8672 44.1671 34.3297 53.3304 34.4779 53.7018C34.626 54.0731 43.7389 56.8397 52.5831 52.0399C56.5468 49.8675 58.0008 45.2812 56.1486 41.8369Z" fill="#2BD67B" />
-  <Path d="M28.1625 5.6665H27.2368V81.0339H28.1625V5.6665Z" fill="#0C152E" />
-  <Path d="M42.374 49.2563L27.48 56.3174L27.8759 57.1567L42.77 50.0957L42.374 49.2563Z" fill="#0C152E" />
-  <Path d="M18.0561 24.2885L17.401 24.9448L27.3713 34.9469L28.0265 34.2906L18.0561 24.2885Z" fill="#0C152E" />
-  <Path d="M38.8861 1.04316C38.594 0.722463 38.2399 0.464678 37.8453 0.285462C37.4507 0.106246 37.0239 0.00932478 36.5907 0.000640343C36.1576 -0.0080441 35.7273 0.0716866 35.3259 0.234941C34.9244 0.398195 34.5603 0.641577 34.2556 0.95031C33.2448 1.83956 32.4241 2.92492 31.8431 4.1409C31.2622 5.35687 30.933 6.67825 30.8754 8.02529C32.2135 8.16272 33.5654 8.02351 34.8477 7.61627C36.13 7.20904 37.3154 6.54245 38.3305 5.65769C39.0145 5.11961 39.4573 4.33117 39.5615 3.46579C39.6657 2.60041 39.4228 1.72898 38.8861 1.04316Z" fill="#2BD67B" />
-  <Path d="M33.7274 4.59109L27.3292 11.0051L27.9841 11.6616L34.3823 5.2476L33.7274 4.59109Z" fill="#0C152E" />
-  <Path d="M54.4268 96.7793H0L0.407485 96.083C3.12617 91.3657 7.0353 87.4485 11.7422 84.7252C16.4491 82.0018 21.788 80.5681 27.2226 80.5681C32.6572 80.5681 37.9962 82.0018 42.7031 84.7252C47.41 87.4485 51.3191 91.3657 54.0378 96.083L54.4268 96.7793ZM1.63918 95.8509H52.8339C50.1461 91.4654 46.382 87.8436 41.9008 85.3312C37.4196 82.8188 32.3709 81.4997 27.2365 81.4997C22.1022 81.4997 17.0534 82.8188 12.5722 85.3312C8.09106 87.8436 4.32696 91.4654 1.63918 95.8509Z" fill="#0C152E" />
-</Svg>
-);
+export default ({ theme }) =>
+  theme === themes.dark ? (
```

Comment:
please avoif to make dublicate code
add fill as variable and change it based on the color
```suggestion
  const fill = theme === themes.dark ? '#9999A0 : '#2BD67B'
```